### PR TITLE
Fix typos in README

### PR DESCRIPTION
**Pull request by rounakbanik**
There were some minor typos I've fixed in the README.
Great project. :)

### CODE REVIEW

File: README.rst [hunk 1]
```diff
@@ -32,7 +32,7 @@ if you want to use these.
 
 
 
-Developement and Instant runnimg
+Development and Instant runnimg
```
Comment (willingc) [inline README.rst]: @rounakbanik Thanks. I noticed one more small typo in the word ``running``.

**Comment by willingc** [comment]
Thanks @rounakbanik. We appreciate the PR. I found one more small typo. Happy to merge after that correction 👍

**Comment by Carreau** [comment]
> Thanks @rounakbanik. We appreciate the PR. I found one more small typo. Happy to merge after that correction 👍

@willingc , what do you think of merging and fix the remaining typo in a separate PR ?

**Comment by willingc** [comment]
Sure. Happy to merge. @rounakbanik, thanks for the contribution. :cookie:

**Pull request merged**